webui/src/lib/api/index.js

```diff
@@ -27,9 +27,10 @@ class LocalCache {
 
 const cache = new LocalCache();
 
-export const linkToPath = (repoId, branchId, path) => {
+export const linkToPath = (repoId, branchId, path, presign=false) => {
     const query = qs({
         path: path,
+        presign: presign
```

Comment:
```suggestion
        path,
        presign,
```
Shorthand if key and values have the same name  + trailing comma.
Not a must. Up to you.

Reply:
done